product/dialogs/miq_dialogs/miq_provision_redhat_dialogs_clone_to_vm.yaml

```diff
@@ -501,15 +501,24 @@
       :description: Hardware
       :fields:
         :disk_format:
+          :values:
+            cow: qcow2
+            raw: raw
+            default: As in template
```

Comment:
@borod108 I think `Same as in template` or `Same as source template` would be a better description